Source/Relativity.Transfer.SDK.Samples.Core/Configuration/Configuration.cs

```diff
@@ -40,7 +48,9 @@ internal record Configuration(
 			new SourceAndDestinationConfiguration(string.Empty, string.Empty),
 			new SourceAndDestinationConfiguration(string.Empty, string.Empty),
 			new SourceAndDestinationConfiguration(string.Empty, string.Empty),
-			new SourceAndWorkspaceIdConfiguration(string.Empty, -1));
+			new SourceAndWorkspaceIdConfiguration(string.Empty, -1),
```

Comment:
-1 means workspace id?

Reply:
yes, this is default WorkspaceId (admin)